docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -0,0 +1,56 @@
+[role="xpack"]
+[[alerting-troubleshooting]]
+== Alerting Troubleshooting
+
+Use the information in this section to troubleshoot common problems and find answers for frequently asked questions.
+
+For issues that you cannot fix yourself … we’re here to help.
+If you are an existing Elastic customer with a support contract, please create a ticket in the
+https://support.elastic.co/customers/s/login/[Elastic Support portal].
+Or post in the https://discuss.elastic.co/[Elastic forum].
+
+
+[float]
+[[alerts-small-check-interval-run-late]]
+=== Alerts with Small Check Intervals Run Late
+
+*Symptoms*:
+
+Alerts are scheduled to run every 2 seconds but they seem to be running too late
+
+*Resolution*:
+
+As described under <<alerting-background-tasks>>, Alerts are ran as background tasks at a cadence defined by their *check interval*.
+When an Alert *check interval* is smaller than the Task Manager <<task-manager-settings,`poll_interval`>> the alert will inevitably run late.
+
+This issue can be addressed by either tweaking the <<task-manager-settings,{kib} Task Manager settings>> or increasing the *check interval* of the Alerts in question.
+
+For details on resolving this issue please see <<task-manager-health-scheduled-tasks-small-schedule-interval-run-late>>.
+
+
+[float]
+[[scheduled-alerts-run-late]]
+=== Alerts Run Late
+
+*Symptoms*:
+
+Scheduled Alerts run at an inconsistent cadence, often running late.
+
+Actions run long after the status of an Alert changes, notifying us of the change too late.
+
+*Resolution*:
+
+As described under <<alerting-background-tasks>>, Alerts and Actions are ran as background tasks ran by each {kib} instance at a default rate of 10 tasks every 3 seconds.
+
+This means that if many alerts or actions have been scheduled to run at the same time, pending tasks will queue in {es}. Each {kib} instance then polls for pending tasks at a rate of up to 10 tasks at a time, at 3 second intervals. As Alerts and Actions are backed by tasks, it is possible for pending tasks in the queue to exceed this capacity and run late as a result.
```

Comment:
```suggestion
If many alerts or actions are scheduled to run at the same time, pending tasks will queue in {es}. Each {kib} instance then polls for pending tasks at a rate of up to ten tasks at a time, at three second intervals. Because alerts and actions are backed by tasks, it is possible for pending tasks in the queue to exceed this capacity and run late.
```